config/samples/metallb.io_v1alpha1_peer.yaml

```diff
@@ -0,0 +1,16 @@
+
+apiVersion: metallb.io/v1alpha1
+kind: Peer
```

Comment:
maybe `BGPPeer` ?

Reply:
I suggested BGPNeighbor below I will close this comment and we can iterate using the 2nd comment below on API definition